docs/versioned_docs/version-0.2.x/sql/1-operators.md

```
@@ -0,0 +1,596 @@
+# Operators and Literals
+
+## Numerical Operators
+
+- [+ (plus)](#op_plus)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)